samples/java-eventsourced-shopping-cart/src/main/java/com/example/shoppingcart/domain/ShoppingCartTestKit.java

```diff
@@ -0,0 +1,53 @@
+package com.example.shoppingcart;
+
+import com.example.shoppingcart.domain.ShoppingCartDomain;
+import com.example.shoppingcart.ShoppingCartApi;
+import com.google.protobuf.Empty;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.Queue;
+import com.akkaserverless.javasdk.testkit.Result;
+
+// Q Would we need ShoppingCartImpl as parameter constructor? 
```

Comment:
true. better do it ourselves

Reply:
>  Since they are both generated we can just call `new ShoppingCart`

This is the implementation by the user, right? I think it'd make sense to have the user instantiate that: that way they can pass any constructor parameters they like (i.e. settings, ...).

Reply:
ahah, @raboof already passed here to say that. Thanks!